.baseline/checkstyle/checkstyle-suppressions.xml

```diff
@@ -21,4 +21,7 @@
     <!-- Generated code should not be subjected to checkstyle. -->
     <suppress files="[/\\].*[/\\]generated_src[/\\]" checks="." />
     <suppress files="[/\\].*[/\\]generated_testSrc[/\\]" checks="." />
+
+    <!-- Exception for SunPkcs1Reader -->
+    <suppress files="SunPkcs1Reader.java" checks="IllegalImport" />
```

Comment:
Not sure if it's possible -- will play around with it a bit more, but the warning/error is on the import (which can't be annotated directly), and as I recall adding `@SuppressWarnings` to the class itself didn't work. Can try adding at package-level to see if that works, though.